src/content/lodash/array/pull/notes.md

```diff
@@ -0,0 +1,3 @@
+Removes all given values from <code>array</code>.
+
+Resources: [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter)
```

Comment:
Could you please use [Array#filter] instead of [filter]?

Reply:
Probably I made some mistake. Original [_.pull](https://lodash.com/docs/4.17.15#pull) mutates an array. My function – doesn't. I'll update my code and come back late. Sorry for disturbing you.

Reply:
I hadn't notice it, good spot!

Reply:
I would recommend using the example that is in the lodash docs to avoid missing implementation details.